### PR TITLE
chore: add copyright headers

### LINKED ISSUE #1628: Add copyright headers for X
Each source file that we ship should have the following license banner:

```
/**
 * Copyright IBM Corp. 2016, 2018
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 */
```

### PR DESCRIPTION
**Pull request by emyarod**
Closes #1628

This PR adds copyright headers to all source files

### CODE REVIEW

File: src/components/accordion/_accordion.scss [hunk 1]
```diff
@@ -1,3 +1,9 @@
+//
```
Comment (joshblack) [inline src/components/accordion/_accordion.scss:1]: Should be able to use the same docblock comment syntax in scss, right? The `/**` kind

Comment (emyarod) [reply src/components/accordion/_accordion.scss:1]: yeah I think the transpiled output will preserve the comments if we use that syntax right? Do we want that as well or would it be sufficient to have the header in the source file only

Comment (emyarod) [reply src/components/accordion/_accordion.scss:1]: I tested and confirmed that the `/** */` comments will remain in the output stylesheet

also noted in the documentation https://sass-lang.com/documentation/file.SCSS_FOR_SASS_USERS.html#Comments

Comment (joshblack) [reply src/components/accordion/_accordion.scss:1]: Does the output style from sass impact things if the built/distributed CSS is minified anyways?

Comment (emyarod) [reply src/components/accordion/_accordion.scss:1]: I guess not but the unminified one would be larger if you were to build it. for what it's worth it looks like material components uses the `//` comment styles for their SCSS as well, for similar reasons I imagine

Comment (joshblack) [reply src/components/accordion/_accordion.scss:1]: Sounds good, will make sure to update in elements then 😄

**Review by joshblack — APPROVED** [review]

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @emyarod!

**Pull request merged**